File: android/testData/projects/jpsWithQualifiedNames/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import android.app.Activity;
import android.os.Bundle;

public class MainActivity extends Activity {

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    new com.example.justlib.JustClass();
    setContentView(R.layout.activity_main);
  }
}

File: android/integration/testData/minnativeapp-apk/minnativeapp/src/main/java/com/example/minapp/MainActivity.kt
package com.example.minnativeapp

import android.app.Activity
import android.os.Bundle
import android.util.Log
import android.widget.TextView

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val label = TextView(this)
        label.setText("Hello Minimal World!")
        Log.w("MainActivity", "Hello Minimal World!")
        setContentView(label)
        Log.w("MainActivity", helloFromJni())
        Log.w("MainActivty", "After breakpoint")
    }

    private external fun helloFromJni() : String

    companion object {
        init {
            System.loadLibrary("minnativeapp")
        }
    }
}
